packages/core/data-transfer/lib/providers/local-strapi-destination-provider/index.ts

```diff
@@ -95,4 +95,30 @@ class LocalStrapiDestinationProvider implements IDestinationProvider {
 
     return mapSchemasValues(schemas);
   }
+
+  async getConfigurationStream(): Promise<Writable> {
+    if (!this.strapi) {
+      throw new Error('Not able to stream Configurations. Strapi instance not found');
+    }
+
+    const strapi = this.strapi;
+
+    return new Writable({
+      objectMode: true,
+      async write(config, _encoding, callback) {
```

Comment:
We could add an `IConfiguration` type in the `common-entities.d.ts` file, what do you think?

Ideally, it would look to smth like

```ts
interface IConfiguration<T = unknown> {
  type: 'core-store' | 'webhook';
  value: T;
}
```